packages/cli/interactive.ts

```diff
@@ -261,15 +313,16 @@ const getSelectRenderers = (
 			const active = i === cursor;
 			const isInListOfValues =
 				Array.isArray(value) && value.includes(optionValue);
-			const color = isInListOfValues || active ? blue : dim;
```

Comment:
@RamIdeas was this change intentional? This is causing select values to render differently than they have in previous versions:

new:
![image](https://github.com/cloudflare/workers-sdk/assets/204386/9f8d1238-8737-4616-9be7-554bbc351e1c)

old:
![image](https://github.com/cloudflare/workers-sdk/assets/204386/49074431-f020-4beb-8172-aff690939c53)

We should change this back or make it overridable somehow

Reply:
Ah yeah my bad I made it match the design doc but I think this is where single select va multi select may differ. We should double check with Meaghan. If we want different styles for single vs multi select we can make it overridable in a base implementation and have higher level getSelectRenderers and getMultiSelectRenderers

Reply:
I also wanted to cc you on the acceptDefault option I added to inputPrompt